source/Classes/020.DnsRecordCname.ps1

```diff
@@ -25,6 +27,14 @@ class DnsRecordCname : DnsRecordBase
 
     DnsRecordCname()
```

Comment:
We cannot use the constructor to throw errors, if we do an instance is newer created and the actual error will never be shown. Also it should be possible to create an instance of the class without setting any properties (and then set properties later) which will never be possible now.